### PR TITLE
bump python-dateutil

### PR DESCRIPTION
**Pull request by emlazzarin**

### CODE REVIEW

**Comment by emlazzarin** [comment]
This fixes #310.

File: setup.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import os
 
-from setuptools import setup, find_packages
+from setuptools import find_packages, setup
```
Comment (emlazzarin) [inline setup.py:3]: This is an auto-resorting from `isort`.

**Comment by medvedev1088** [comment]
Thanks @emlazzarin! This should be pretty safe with python-dateutil. Other less reliable libs are locked as they sometimes introduce breaking changes even in minor version or patch upgrades

**Comment by medvedev1088** [comment]
Released in 1.10.1 https://github.com/blockchain-etl/ethereum-etl/releases/tag/v1.10.1

**Pull request merged**